components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/index.tsx

```diff
@@ -118,12 +139,13 @@ const AddAccountModal = (props: Props) => {
 
   return (
     <PopupModal title={title} onClose={onClose}>
-      <TopTabNav
-        tabList={AccountSettingsNavOptions}
-        onSubmit={onChangeTab}
-        selectedTab={tab}
-      />
-
+      {!hideNav && onChangeTab &&
```

Comment:
Why do we need `&& onChangeTab` here?

Reply:
Since `onChangeTab` is possibly `undefined` i had it as a check there to prevent a `TS` error.
Just moved this check into a separate `function`! 😎

https://github.com/brave/brave-core/pull/9658/files#diff-a37d9983af6c7ab7f4acfea81db505c73da9956b18ba5d9d762b9ba103ba99dfR140-R145